src/Bridges/Constraint/geomean_to_relentr.jl

```diff
@@ -0,0 +1,95 @@
+    F = MOI.VectorOfVariables
+    S = MOIU.scalar_type(H)
+    G = MOIU.promote_operation(vcat, T, T, S, MOIU.promote_operation(+, T, S, MOI.SingleVariable))
+    return GeoMeantoRelEntrBridge{T, F, G, H}
+end
+
+# Attributes, Bridge acting as a model
+MOI.get(bridge::GeoMeantoRelEntrBridge, ::MOI.NumberOfVariables) = 1
+MOI.get(bridge::GeoMeantoRelEntrBridge, ::MOI.ListOfVariableIndices) = [bridge.y]
+MOI.get(bridge::GeoMeantoRelEntrBridge{T, F}, ::MOI.NumberOfConstraints{F, MOI.Nonnegatives}) where {T, F} = 1
+MOI.get(bridge::GeoMeantoRelEntrBridge{T, F, G}, ::MOI.NumberOfConstraints{G, MOI.RelativeEntropyCone}) where {T, F, G} = 1
+MOI.get(bridge::GeoMeantoRelEntrBridge{T, F}, ::MOI.ListOfConstraintIndices{F, MOI.Nonnegatives}) where {T, F} = [bridge.nn_index]
+MOI.get(bridge::GeoMeantoRelEntrBridge{T, F, G}, ::MOI.ListOfConstraintIndices{G, MOI.RelativeEntropyCone}) where {T, F, G} = [bridge.relentr_index]
+
+# References
+function MOI.delete(model::MOI.ModelLike, bridge::GeoMeantoRelEntrBridge)
+    MOI.delete(model, bridge.relentr_index)
+    MOI.delete(model, bridge.nn_index)
+    MOI.delete(model, bridge.y)
+end
+
+# Attributes, Bridge acting as a constraint
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintFunction, bridge::GeoMeantoRelEntrBridge{T, F, G, H}) where {T, F, G, H}
+    relentr_func = MOIU.eachscalar(MOI.get(model, MOI.ConstraintFunction(), bridge.relentr_index))
+    d = div(length(relentr_func) - 1, 2)
+    u_func = MOIU.remove_variable(MOIU.operate(-, T, relentr_func[end], MOI.SingleVariable(bridge.y)), bridge.y)
+    w_func = relentr_func[2:(1 + d)]
+    return MOIU.convert_approx(H, MOIU.operate(vcat, T, u_func, w_func))
+end
+MOI.get(model::MOI.ModelLike, ::MOI.ConstraintSet, bridge::GeoMeantoRelEntrBridge) = MOI.GeometricMeanCone(1 + div(MOI.dimension(MOI.get(model, MOI.ConstraintSet(), bridge.relentr_index)) - 1, 2))
+MOI.supports(::MOI.ModelLike, ::Union{MOI.ConstraintPrimalStart, MOI.ConstraintDualStart},
+    ::Type{<:GeoMeantoRelEntrBridge}) = true
+function MOI.get(model::MOI.ModelLike, attr::Union{MOI.ConstraintPrimal, MOI.ConstraintPrimalStart}, bridge::GeoMeantoRelEntrBridge)
+    relentr_primal = MOI.get(model, attr, bridge.relentr_index)
+    d = div(length(relentr_primal) - 1, 2)
+    y_val = MOI.get(model, attr, bridge.nn_index)[1]
+    u_val = sum(relentr_primal[(2 + d):end]) / d - y_val
+    return vcat(u_val, relentr_primal[2:(1 + d)])
+end
+function MOI.set(model::MOI.ModelLike, attr::MOI.ConstraintPrimalStart, bridge::GeoMeantoRelEntrBridge, value)
+    u_val = value[1]
+    u_pos = -min(zero(u_val), u_val)
+    d = length(value) - 1
+    MOI.set(model, MOI.VariablePrimalStart(), bridge.y, u_pos)
+    MOI.set(model, attr, bridge.nn_index, [u_pos])
+    MOI.set(model, attr, bridge.relentr_index, vcat(0, value[2:end], fill(u_pos + u_val, d)))
+    return
+end
+# Given a is dual on y >= 0 and (b, c, d) is dual on RelativeEntropyCone constraint,
+# dual on (u, w) in GeometricMeanCone is (-a, c).
```

Comment:
I have checked and I have the same thing with (2.8) of https://drive.google.com/file/d/1Y8kvUsHmvWSvgnEBS8T7GI6ofvuWng6U/view?usp=sharing
I would add in the comment that we have `sum(d) = -a` so we could also use `sum(d)` instead of `-a`.

Reply:
See updated comment